app/models.py

```diff
@@ -423,6 +461,7 @@ class Session(BaseModel):
     created_at: datetime = Field(default_factory=datetime.utcnow)
     events: List[Event] = []
     has_quiz_ended: bool = False
+    metrics: Optional[SessionMetrics] = None
```

Comment:
why is this needed? when session starts there will be no metrics to report right?